src/Fixer/Phpdoc/AlignMultilineCommentFixer.php

```diff
@@ -47,7 +47,7 @@ final class AlignMultilineCommentFixer extends AbstractFixer implements Configur
     use ConfigurableFixerTrait;
 
     /**
-     * @var null|list<int>
+     * @var null|non-empty-list<int>
      */
     private ?array $tokenKinds = null;
```

Comment:
kind like `kind` (id|string)? then docblock is wrong.
kind like one-of-old-meanings `id` ? then let's rename the property